assets/js/main.js

```diff
@@ -61,4 +63,9 @@ $(function() {
       });
     }
   });
+
+
+  // Required element to turn into an accordion
+  var elm = document.querySelector('.js-accordion');
+  var lol = new accordion.Accordion(elm, {}, { collapseOthers: false, reflectStatic: true });
```

Comment:
I noticed we get some console errors on pages where the accordion isn't present - should we test that the element exists before initializing it here?